.circleci/config.yml

```
@@ -1,5 +1,25 @@
 version: 2.1
 
+parameters:
+  dev-s3-bucket:
+    type: string
+    default: 'cdn.dev.w3w.io'
+  prod-s3-bucket:
+    type: string
+    default: 'cdn.what3words.com'
+
+only-prerelease-tags: &only-prerelease-tags
+  filters:
+    tags:
+      only: /^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9]+)*))?(?:\+[0-9]+)?$/
```

```suggestion
      only: /^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9]+)*)){1}(?:\+[0-9]+)?$/
```
If this is only prerelease then I presume it should have the prerelease suffix